<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndirectBayes</class>
 <widget class="QMainWindow" name="IndirectBayes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Indirect Bayes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="indirectBayesTabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="resNorm">
      <attribute name="title">
       <string>ResNorm</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="Quasi">
      <attribute name="title">
       <string>Quasi</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="Stretch">
      <attribute name="title">
       <string>Stretch</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="JumpFit">
      <attribute name="title">
       <string>JumpFit</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_bottom">
     <item>
      <widget class="QPushButton" name="pbHelp">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbRun">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbManageDirs">
       <property name="text">
        <string>Manage Directories</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
